### ISSUE TITLE
Would it be possible to make img to 3D

### ISSUE THREAD

**Issue by destlaver** [openai/point-e#19]
if I understood the model is trained this way: text -> generated rendering img by GLIDE -> 3D. So in theory it would be possible to generate 3D from an image. My question is if it possible even with images without. shadows like cartoon characters or drawings. Great work btw and thank you for making it open source

**Comment by jarble** [comment]
I found a demo of Point-E [here](https://huggingface.co/spaces/anzorq/point-e_demo) that seems to have this capability.

**Comment by destlaver** [comment]
thanks, I hope they will improve the model soon

**Comment by alsolh** [comment]
Yes, check this video https://youtube.com/shorts/zmLYaGek81Y?feature=share
